scss/detail.scss

```diff
@@ -0,0 +1,36 @@
+.book-container {
+  font-family: 'Roboto', sans-serif;
+  padding: 15px;
+}
+
+.column {
+  display: flex;
+  flex-direction: column;
+}
+
+.row {
+  display: flex;
+}
+
+.book-title {
+  font-size: 32px;
+  font-weight: bold;
+}
+
+.book-gender {
+  color: $gray;
+  font-size: 24px;
+  font-weight: bold;
+  margin-left: 8px;
+  padding-bottom: 18px;
```

Comment:
If you need to separate this line from the others you should use `margin-bottom` instead. Padding is for separating inner elements.